assembler.py

```diff
@@ -36,7 +36,7 @@ def verify_ram_content():
 
 def hexify_ram_content():
     for i in range(4096):
-        RAM[i] = f'{int(RAM[i], 2):02X}'
+        RAM[i] = f'{int(RAM[i], 16):02X}'
```

Comment:
Revert this. RAM content in this instance should be in binary

Reply:
okay, but then you have to check very well the testers for the assembler, because this was one of the errors it gave out. Whether it's a quick fix or not, it was something needed to at least manage to run the tests the prof gave us.